modules/nodes-nodes-working-adding.adoc

```diff
@@ -0,0 +1,160 @@
+// Module included in the following assemblies:
```

Comment:
this entire section is obsolete for 4.0.  nodes are added to the cluster using `MachineSets` in 4.0.